tests/feature_tests/upgrade/upgrade_vars.sh

```diff
@@ -6,10 +6,10 @@ set -eux
 export CAPIRELEASE_HARDCODED="v1.2.99"
 
 export CAPI_VERSION="${CAPI_VERSION:-v1alpha4}"
-export CAPI_REL_TO_VERSION="v1.2.1"
+export CAPI_REL_TO_VERSION="v1.2.2"
 
 export CAPM3_VERSION="${CAPM3_VERSION:-v1alpha5}"
-export CAPM3_REL_TO_VERSION="v1.2.0-rc.1"
+export CAPM3_REL_TO_VERSION="v1.2.0"
```

Comment:
I had to fix the upgrade tests as well since we have now latest release (instead of rc)